### ISSUE TITLE
In eval(),  BatchNorm running_mean is different between different GPUs

### ISSUE THREAD

**Issue by yangkai798** [pytorch/pytorch#13711]
## 🐛 Bug

After train , when test the model, the BatchNorm is different between different GPUs, it does not do broadcast in eval()?

## To Reproduce

1.
1.
1.

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by zou3519** [comment]
Please give us more information /a repro.

**Comment by yangkai798** [comment]
I means after run val after train, in loading a checkpoint , the running_mean is same between different gpus. But when running val after train, the running_mean is different between GPUs.
You can reproduction by running the same image in different gpus.
Or it is a character , not a bug? when you save the checkpoint in rank 0, it just save the rank0 GPU's
BatchNorm parameters

**Comment by mruberry** [comment]
Closing due to age; if you're still encountering this issue on the latest PyTorch please open a new issue with an updated reproduction
